src/app/(BottomTabNavigation)/Profile/EditName/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { router } from 'expo-router';
+import React, { useEffect, useState } from 'react';
+import { Text, View, TouchableOpacity, TextInput } from 'react-native';
+
+import styles from './styles';
+import {
+  getCurrentUserInfo,
+  updateCurrUserName,
+} from '../../../../supabase/queries/auth';
+
+function EditNameScreen() {
+  const [currSession, setCurrSession] = useState<any>(null);
```

Comment:
Same comment as above but let's use `User` type and set it to a default `userInstance`